lib/web/apiserver.go

```diff
@@ -590,13 +590,8 @@ func (h *Handler) bindDefaultEndpoints() {
 	h.DELETE("/webapi/sites/:site/locks/:uuid", h.WithClusterAuth(h.deleteClusterLock))
 
 	// active sessions handlers
-	h.GET("/webapi/sites/:site/connect", h.WithClusterAuth(h.siteNodeConnect))  // connect to an active session (via websocket)
-	h.GET("/webapi/sites/:site/sessions", h.WithClusterAuth(h.siteSessionsGet)) // get active list of sessions
-	// TODO POSTS to `/webapi/sites/:site/sessions` should no longer be required
-	// but this endpoint is still used by the UI. When time allows evaluate the
-	// removal of this handler and the associated methods here and in the UI.
-	h.POST("/webapi/sites/:site/sessions", h.WithClusterAuth(h.siteSessionGenerate)) // create active session metadata
```

Comment:
@kimlisa I'm trying to get rid of this endpoint because I don't see it being used on the frontend, but it looks like it still is being used in this test:
https://github.com/gravitational/teleport/blob/c144e3a115ab233a502afc309c3739190f5cffcd/lib/web/apiserver_test.go#L2082-L2086

Is this test still necessary? And if so, is there a better way to rewrite it?

Reply:
i don't see this endpoint being used in the UI anymore? so the endpoint and test is not needed. it looks like we are generating new session through the websocket connection now (link below) and contains the necessary test, so no extra test needed: https://github.com/gravitational/teleport/pull/18523/files#diff-7ccc0b653c1df2f24c01f64ed2525daa493092ad3c97df16de3c36e7f05e3911

Reply:
while you're at it, can you delete this (i don't see it used): https://github.com/gravitational/teleport/blob/e1f3fb8e4db82ffe14c91f66a67b489c863f92c6/web/packages/teleport/src/config.ts#L162